backend/src/migrations/20230323142236_add_wallets.ts

```diff
@@ -0,0 +1,85 @@
+import { type Knex } from 'knex';
+
+const WALLETS_TABLE = 'wallet';
+const WALLETS_USERS_TABLE = 'wallets_users';
+const FOREIGN_TABLE_NAME = 'users';
+
+const uuid = 'uuid_generate_v4()';
+
+const WalletsColumnName = {
+    ID: 'id',
+    CURRENCY_ID: 'currency_id',
+    NAME: 'name',
+    BALANCE: 'balance',
+    OWNER_ID: 'owner_id',
+    CREATED_AT: 'created_at',
+    UPDATED_AT: 'updated_at',
+};
+
+const WalletsUsersColumnName = {
+    ID: 'id',
+    USER_ID: 'user_id',
+    WALLET_ID: 'wallet_id',
+};
+
+const RelationRule = {
+    CASCADE: 'CASCADE',
+    SET_NULL: 'SET NULL',
+};
+function up(knex: Knex): Promise<void> {
+    return knex.schema
+        .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
+        .dropTableIfExists(WALLETS_USERS_TABLE)
+        .dropTableIfExists(WALLETS_TABLE)
```

Comment:
Why did you decide to drop the table and create a new one instead of editing the existing one?

Reply:
i decided to do it because Vasyl created table with a name 'wallet' whilst we need table 'wallets'

Reply:
I fixed it as you said. I thought Vasyl had different name but i was wrong